docs/recipes/htmlhint.md

```diff
@@ -30,25 +30,48 @@ Read [gulp-htmlhint's docs][api] to find out how you can pass options to HTMLHin
 
 HTMLHint should run on serve and build.
 
-```js
-gulp.task('html', ['htmlhint', 'styles', 'scripts'], () => {
+```diff
+ const build = series(
+   clean,
+   parallel(
+     lint,
+-    series(parallel(styles, scripts), html),
++    series(parallel(htmlhint, styles, scripts), html),
+     images,
+     fonts,
+     extras
+   ),
+   measureSize
+ );
 ```
 
-```js
-gulp.task('serve', () => {
-  runSequence(['clean', 'wiredep'], ['htmlhint', 'styles', 'scripts', 'fonts'], () => {
+```diff
+ let serve;
+ if (isDev) {
+-  serve = series(clean, parallel(styles, scripts, fonts), startAppServer);
++  serve = series(clean, parallel(htmlhint, styles, scripts, fonts), startAppServer);
+ } else if (isTest) {
+   serve = series(clean, scripts, startTestServer);
+ } else if (isProd) {
+   serve = series(build, startDistServer);
+ }
 ```
 
 ### 4. Run the task on each HTML change
 
 In the `serve` task add the following line to run `htmlhint` every time a HTML file in the `app` directory changes:
 
 ```diff
-+gulp.watch('app/**/*.html', ['htmlhint']);
- gulp.watch('app/styles/**/*.scss', ['styles']);
- gulp.watch('app/scripts/**/*.js', ['scripts']);
- gulp.watch('app/fonts/**/*', ['fonts']);
- gulp.watch('bower.json', ['wiredep', 'fonts']);
+   watch([
+     'app/*.html',
+     'app/images/**/*',
+     '.tmp/fonts/**/*'
+-  ]).on('change', server.reload);
++  ], htmlhint).on('change', server.reload);
```

Comment:
Sorry, I totally missed the more obvious thing 🤦‍♂️ I'd rather create a separate watcher which watches only `app/**/*.html` files and runs `htmlhint` instead of running it here. Because images and fonts can't affect the results of htmlhint, right?

Reply:
(By "watcher" I just mean adding another `gulp.watch`.)

Reply:
I totally agree! 👍
It's now fixed.